<?xml version="1.0"?>
<launch>

<param name="active" type="int" value="0" />

<param name="des_pos_x" type="double" value="1" />
<param name="des_pos_y" type="double" value="1" />


<node pkg="final_assignment" type="UI.py" name="UI" required="true" launch-prefix="xterm -e" />
<node pkg="final_assignment" type="go_to_desired_pos.py" name="go_to_desired_pos" required="true" launch-prefix="xterm -e" />
<node pkg="final_assignment" type="my_teleop_twist_keyboard.py" name="my_teleop_twist_kb" required="true" launch-prefix="xterm -e" /> 



</launch>